{"name": "Ручка /api/v1/courier/login", "status": "passed", "description": "Пробуем залогиниться, используя некорректные данные, ожидаем получить ошибку 400 и текст 'Недостаточно данных для входа'", "parameters": [{"name": "login", "value": "None"}, {"name": "password", "value": "'None'"}], "start": 1710612252953, "stop": 1710612253001, "uuid": "8e998f29-b05a-4e3a-9379-b02735234352", "historyId": "ff482f7902d5ff98a4ffe78c815c7276", "testCaseId": "14eb7ed67efb59bc0210cf649d3f4709", "fullName": "test_api.tests.test_login_courier.TestLoginCourier#test_login_courier_empty_fields_error", "labels": [{"name": "parentSuite", "value": "test_api.tests"}, {"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "LAPTOP-KS9V29SF"}, {"name": "thread", "value": "22020-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_api.tests.test_login_courier"}]}